multicluster/test/e2e/main_test.go

```diff
@@ -83,9 +95,55 @@ func testMain(m *testing.M) int {
 	rand.Seed(time.Now().UnixNano())
 
 	ret := m.Run()
+	if ret != 0 {
+		return ret
```

Comment:
Probably better add a log like `log.Println("Failed to run default Multi-cluster E2E tests")` before return, so we can know which part of tests failed quicker.